spec/components/proposal_details/summary_component_spec.rb

```diff
@@ -35,15 +35,15 @@
   describe "#auto_answered?" do
     context "when question was auto answered" do
       it "returns true" do
-        expect(summary_component.send(:auto_answered?)).to eq(true)
+        expect(summary_component.send(:auto_answered?)).to be(true)
```

Comment:
I didn't know about `be` until just the other day. Much more human-readable 🤩

Reply:
Yeah I totally agree, I also prefer the `be` compare to `eq` :D